src/main/java/org/diatliuk/bookstore/dto/cart/item/CreateCartItemRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package org.diatliuk.bookstore.dto.cart.item;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class CreateCartItemRequestDto {
+    @NotNull
+    private Long bookId;
+
+    @NotNull
+    @Min(value = 1)
```

Comment:
```suggestion
    @Min(1)
```
you can omit specifying `value`, because if you pass only 1 parameter to the annotation `value` is considered as default